src/test/COMPILER_TESTS.md

```diff
@@ -53,6 +53,7 @@ be compiled or run.
 * `ignore-pretty` will not compile the pretty-printed test (this is done to test the pretty-printer, but might not always work)
 * `ignore-test` always ignores the test
 * `ignore-lldb` and `ignore-gdb` will skip a debuginfo test on that debugger.
+* `ignore-gdb-version` can be used to ignore the test when certain gdb versions are used
```

Comment:
hmm I feel like this document ought to be removed and just become a pointer to

https://rust-lang-nursery.github.io/rustc-guide/tests/intro.html

(cc @mark-i-m, thoughts?)

Reply:
and this PR would be resubmitted at rustc-guide, natch

Reply:
is there anything in here that is not present in the guide? I guess we have to check

Reply:
I looked and the guide seems like a superset of what's in this file.